ARKitExample.xcodeproj/project.pbxproj

```diff
@@ -13,6 +13,7 @@
 		1C0964A21F3CF21B008F2C19 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 1C0964A01F3CF21B008F2C19 /* Main.storyboard */; };
 		1C0964A41F3CF21B008F2C19 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 1C0964A31F3CF21B008F2C19 /* Assets.xcassets */; };
 		1C0964A71F3CF21B008F2C19 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 1C0964A51F3CF21B008F2C19 /* LaunchScreen.storyboard */; };
+		D2842487EAFB535D18D44721 /* libPods-TwilioVideo-ARKitExample.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 69D6C16463979EFE635B507C /* libPods-TwilioVideo-ARKitExample.a */; };
```

Comment:
Unexpected that these project files were modified. Maybe because I'm using Xcode 11.4 or maybe I'm using a different pod version. Looks like Podfile.lock is not in this repo.

Reply:
I have reverted the project file changes caused by pod install.